src/mongo/db/repl/replication_coordinator_impl_test.cpp

```diff
@@ -5511,21 +5494,14 @@ TEST_F(
     OpTimeWithTermOne time(100, 2);
 
     // 3 nodes waiting for time
-    WriteConcernOptions writeConcern;
-    writeConcern.wTimeout = WriteConcernOptions::kNoTimeout;
-    writeConcern.wNumNodes = 3;
+    WriteConcernOptions writeConcern(
+        3, WriteConcernOptions::SyncMode::UNSET, WriteConcernOptions::kNoTimeout);
 
     ReplicationAwaiter awaiter(getReplCoord(), getServiceContext());
     awaiter.setOpTime(time);
     awaiter.setWriteConcern(writeConcern);
     awaiter.start();
 
-    ReplicationAwaiter awaiterJournaled(getReplCoord(), getServiceContext());
```

Comment:
This awaiter originally returned an error because the `WriteConcernOptions` above was reused, which is a bug. In `ReplSetConfig::checkIfWriteConcernCanBeSatisfied` if a `wMode` of "majority" is used then we check the `wNumNodes` of the write concern to determine if there are enough nodes to satisfy the write concern. A properly constructed majority `WriteConcernOptions` has `wNumNodes: 0` so this code would now just block indefinitely waiting for an actual majority. I chose to remove the test since I didn't think it was actually testing the correct functionality